docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/MulticlassClassification.ttinclude

```diff
@@ -44,7 +51,12 @@ namespace Samples.Dynamic.Trainers.MulticlassClassification
             var options = new <#=TrainerOptions#>;
 
             // Define the trainer.
-            var pipeline = mlContext.MulticlassClassification.Trainers.<#=Trainer#>(options);
+            var pipeline = 
+			        // Convert the string labels into key types.
```

Comment:
a line not aligned.

Reply:
It's done on purpose, so that we can comment before the two estimators added to the pipeline.

---
In reply to: [275097557](https://github.com/dotnet/machinelearning/pull/3322#discussion_r275097557) [](ancestors = 275097557)